Added link to proxy section when make template-example fails

Neha-Sinha2305 opened the pull request:
I went through the [Makefile](https://github.com/fredrikhgrelland/terraform-nomad-redash/blob/c59576242e47ccaed98d083a073aa3eb50be8c6f/Makefile#L49) and i see that we already have the error message regarding missing certificate added to the update-box block. The user is presented with the link to [proxy section]( https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#if-you-are-behind-a-transparent-proxy) in the README file for the following commands:

1. ```make dev```
2.  ```make up ```
3.  ```make test ```,
Since, all the above have a dependency on 'update-box' so the user gets the error message as expected.

The only command that does not present the user with the link to the proxy section is :

1.  ```make template-example ``` as it does not depend on the  'update-box'

I am going ahead and adding the error message to the following line and i guess that should be good enough: https://github.com/fredrikhgrelland/terraform-nomad-redash/blob/c59576242e47ccaed98d083a073aa3eb50be8c6f/Makefile#L36

pdmthorsrud reviewed (commented):
Could you possibly add `update-box` as a prereq for the `template-example` target, instead of adding the print to the recipe of the `template-example`?

Neha-Sinha2305 commented:
> Could you possibly add `update-box` as a prereq for the `template-example` target, instead of adding the print to the recipe of the `template-example`?

do we need to run the 'update-box' when we run the tests in CI?https://github.com/fredrikhgrelland/terraform-nomad-redash/blob/c59576242e47ccaed98d083a073aa3eb50be8c6f/Makefile#L32

pdmthorsrud commented:
> > Could you possibly add `update-box` as a prereq for the `template-example` target, instead of adding the print to the recipe of the `template-example`?
> 
> do we need to run the 'update-box' when we run the tests in CI?
> 
> https://github.com/fredrikhgrelland/terraform-nomad-redash/blob/c59576242e47ccaed98d083a073aa3eb50be8c6f/Makefile#L32

No, but does it hurt if we do?

Neha-Sinha2305 commented:
> > > Could you possibly add `update-box` as a prereq for the `template-example` target, instead of adding the print to the recipe of the `template-example`?
> > 
> > 
> > do we need to run the 'update-box' when we run the tests in CI?
> > https://github.com/fredrikhgrelland/terraform-nomad-redash/blob/c59576242e47ccaed98d083a073aa3eb50be8c6f/Makefile#L32
> 
> No, but does it hurt if we do?

No, it doesn't. 
I have changed the code now and added update-box as a prereq for the template-example target.

pdmthorsrud commented:
> No, it doesn't.
> I have changed the code now and added update-box as a prereq for the template-example target.

Nice! Personally I feel like that is more readable. Open to be wrong though!

pdmthorsrud reviewed (approved):
LGTM
